script/hassfest/translations.py

```diff
@@ -24,6 +24,11 @@
 RE_TRANSLATION_KEY = re.compile(r"^(?!.+[_-]{2})(?![_-])[a-z0-9-_]+(?<![_-])$")
 RE_COMBINED_REFERENCE = re.compile(r"(.+\[%)|(%\].+)")
 RE_PLACEHOLDER_IN_SINGLE_QUOTES = re.compile(r"'{\w+}'")
+RE_URL = re.compile(
+    r"(((homeassistant|ftp|ftps|scp|http|https|mqtt|mqtts|socket|socks5):\/\/|www\.)"
```

Comment:
why the homeassistant protocol?

Reply:
It can be used for internal URL's. Not sure it was used though.

Reply:
Removed it. Lets see if the the tests pass.